yarn.lock

```diff
@@ -29589,7 +29632,7 @@ snapdragon@^0.8.1:
     source-map-resolve "^0.5.0"
     use "^3.1.0"
 
-snarkjs@0.4.10, snarkjs@^0.4.10:
+snarkjs@^0.4.10:
```

Comment:
we need snarkjs hard pinned to 0.4.10

Reply:
otherwise things will break with the zk redeem flow

Reply:
Good catch. @hnguyen1179 , wasn't there something we did to handle this with another package so that it would always resolve to a fixed version even if yarn.lock changed? I can't remember what we did.

Reply:
Oh, @sentilesdal, it looks like we already did it. The `resolutions` field in the top level `package.json` has the version pinned, so even if this line changes, it will still resolve to the version in the `resolutions` field. You can verify this by looking at the line below this (https://github.com/element-fi/frontend-monorepo/pull/363/files/b1bc5f4c27748da1de0257fb1408f353c1f375a3#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR29636)